authors/wubwub/05-SegWit2x/hr_HR/final.md

```diff
@@ -0,0 +1,181 @@
+Segwit? Segwit2x? Fork? NYA agreement? Sve su to pojmovi koje ste možda čuli u proteklih nekoliko mjeseci, a možda ne znate što znače? U ovom članku ćemo Vam pojasniti spomenute pojmove i što se krije iza njih.
+
+*Ako ste novak u kriptovalutama, da bi razumjeli problematiku ovoga, preporučujemo da pročitate [uvod u blockchain][blockchain].*
+
+
+## Problematika
+
+Svaki blockchain ima istu slabost, a to je skalabilnost. Kad blockchain dođe do točke zasićenja gdje ima više transakcija nego mreža može procesirati, brzina kojom se transakcije izvršavaju se smanjuje, nešto kao kad se cesta suzi s 4 trake na 2 trake.
+
+Kako se automobili na cesti gužvaju u 2 trake, tako transakcije ne mogu stati u blok i moraju čekati svoj red, no još jedan problem koji se ovdje javlja je taj što tko plati veću naknadu za transakciju, njegova transakcija ima i veći prioritet. To znači ne samo da se mreža uspori zbog puno transakcija, nego i naknade za transakcije rastu.
+
+Sad se vratimo 2 godine u prošlost kad je počela rasprava o skalabilnosti Bitcoina.
+Popularnost Bitcoina je skočila, a s time i stopa transakcija.
+Bitcoin nije mogao procesirati sve transakcije u očekivanom vremenu i gore navedeni scenarij je postao stvarnost.
+
+Drugi problem koji se u međuvremenu pojavio je bug po imenu [ASICBoost].
+
+ASICBoost omogućava veću efektivnost rudarenja. Manja potrošnja električne energije po GH/s (Gigahash/sekunda). Kineski proizvođač opreme za rudarenje, Bitmain imao je ugrađen ASICBoost u rudare koje su oni koristili, a time su zarađivali 30% Bitcoina više, što je otprilike do $100 milijuna godišnje.
+
+
+## Rješenje?
+
+Tu se javljaju problemi jer imamo suprotstavljena dva kampa: 
+
+1. Core developeri (programeri koji rade na Bitcoin protokolu)
+2. Kineski rudari
+
+### 1. Rješenje: 
+
+**Core developeri** žele implementirati Segregated Witness (SegWit u daljnjem tekstu), koje je kompleksnije rješenje, ali isplativije na duže staze. Također postavlja temelje za *Lightning network* koja će dodatno povećati skalabilnost s off-chain transakcijama. 
```

Comment:
Previše tehničkih termina. Objašnjenje segwita prebaciti ovamo. LN objasniti na laički način.